README.md

```diff
@@ -7,7 +7,13 @@
 
 Build development environments from a Dockerfile on Docker, Kubernetes, and OpenShift. Allow developers to modify their environment in a tight feedback loop.
 
-- Supports [`devcontainer.json`](https://containers.dev/) and `Dockerfile`
+- Supports a subset of [`devcontainer.json`](https://containers.dev/) and `Dockerfile`. Currently support is missing for the following `devcontainer.json` features
+
+  - `image`
+  - `build`
+  - `runArgs`
+  - `workspaceFolder`
```

Comment:
We actually support both `image` and `build`!

Reply:
I just pulled it the old commit. Are there things that are there other features that are missing, especially important ones like `customizations` and `mounts`. Looks like lifecycle scripts are supported?